src/Controls/src/Core/Compatibility/Handlers/NavigationPage/iOS/NavigationRenderer.cs

```diff
@@ -1451,7 +1451,7 @@ void UpdateTitleImage(Container titleViewContainer, ImageSource titleIcon)
 				}
 				else
 				{
-					titleIcon.LoadImage(titleIcon.FindMauiContext(), result =>
+					titleIcon.LoadImage(titleIcon.FindMauiContext(true), result =>
```

Comment:
Seems similar to https://github.com/dotnet/maui/pull/14634, @PureWeen Thoughts?

Reply:
I think you can just use the `MauiContext` that's on this class and call that good